frontend-v2/src/index.tsx

```diff
@@ -1,15 +1,10 @@
-import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
 
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
+root.render(<App />);
```

Comment:
I believe we should not remove react strict mode from the code.

I know we get some duplicated sockets and components when we use the strict mode, but this helps getting some errors and bugs on development.

When testing locally, we can just remove the strict mode from the code temporarily or do some `npm run build && npx serve -s build` to deploy production locally (_Strict mode checks are run in development mode only; they do not impact the production build_). 

WDYT?